user_guide_src/source/database/query_builder.rst

```diff
@@ -1608,19 +1608,45 @@ Class Reference
 
         Compiles and executes an ``UPDATE`` statement.
 
-    .. php:method:: updateBatch([$set = null[, $value = null[, $batch_size = 100]]])
+    .. php:method:: updateBatch([$set = null[, $constraints = null[, $batchSize = 100]]])
 
         :param array $set: Field name, or an associative array of field/value pairs
-        :param string $value: Field value, if $set is a single field
-        :param int $batch_size: Count of conditions to group in a single query
+        :param mixed $constraints: The field or fields used as keys to update on.
+        :param int $batchSize: Count of conditions to group in a single query
         :returns:   Number of rows updated or ``false`` on failure
         :rtype:     int|false
 
         Compiles and executes batch ``UPDATE`` statements.
 
```

Comment:
```suggestion
The ``$constraints`` variable takes a comma delimited string of columns, an array, an associative array, or RawSql.
```

I would move this description a little higher. Not sure if it should be put into "note".

Reply:
Done